app/index.js

```diff
@@ -23,7 +23,7 @@ module.exports = generators.Base.extend({
                 type: 'input',
                 name: 'name',
                 message: 'Your project name',
-                default: this.appname // Default to current folder name
+                default: 'react-engine-generator' // Default to current folder name
```

Comment:
What if we prompted the users to see if they'd like to create a new folder? Or at the very least came up with a helper function to turn spaces into underscores/trailcase?

```javascript
if (this.newDirectory) {
   this.destinationRoot(utils.slug(this.projectName));
}
```